src/main/kotlin/app/model/DiffContent.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.model
+
+import java.nio.file.Path
+import java.nio.file.Paths
+
+/**
+ * Per file diff from commit.
+ */
+data class DiffContent(
+        var path: Path = Paths.get(""),
```

Comment:
I think it's fine for now, but in future, we will need to track more info so that extractors could make better decisions. For instance, antlr will need an entire file to parse out, so the extractor will need line numbers that we were added or deleted, not the lines themselves.